website/src/blogs/provisioning-openebs-jiva-volumes-via-csi.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Provisioning OpenEBS Jiva volumes via CSI
+author: Prateek Pandey
+author_info: Contributor and Maintainer @OpenEBS. Software Developer at @mayadata_inc. Open Source Enthusiast
+date: 09-06-2021
+tags: CSI, OpenEBS, Jiva
+excerpt: Container Storage Interface (CSI), is the new model for integrating storage system drivers with container orchestration systems like Kubernetes. This new interface is a major benefit to the container ecosystem as it standardizes the model of integrating storage systems with container orchestration systems.
+not_has_feature_image: false
+---
+
+[Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec), is the new model for integrating storage system drivers with container orchestration systems like [Kubernetes](https://kubernetes.io/). This new interface is a major benefit to the container ecosystem as it standardizes the model of integrating storage systems with container orchestration systems. Specifically for Kubernetes, it frees the storage system driver from being tied to the Kubernetes release schedule due to it being incorporated in the same code base. With CSI, storage system drivers can now be installed asynchronously to container orchestration releases, providing faster bug fixes and features.
+
+The Openebs Jiva CSI driver provides a CSI interface that allows it to manage the lifecycle of OpenEBS Jiva volumes for persistent volumes. OpenEBS has recently released a beta version of the [Jiva CSI driver](https://github.com/openebs/jiva-operator) for one of its Container Attached Storage engines - [Jiva](https://github.com/openebs/jiva).
+
+As of 2.7.0, the supported features are volume provisioning, de-provisioning, volume resize, raw block volume, and volume usage metrics. There are other key differences between the external-provisioner and CSI implementations, are benefits to users, like
+
+- Auto remount
+- Protecting against nodes mounting an empty directory accidentally and re-writing. 
+- Fix for multi-attach errors on onprem k8s clusters
+- Standardised with CSI and further enhancements coming in K8s
+
+This blog will demonstrate how to configure and consume Jiva volumes using the CSI Driver. 
+
+## Prerequisites
+
+1. Kubernetes version 1.18 or higher
+2. iSCSI initiator utils installed on all the worker nodes
+3. You have access to install RBAC components into the openebs namespace.
+4. For resize feature, verify ExpandCSIVolumes and ExpandInUsePersistentVolumes feature gates are enabled on the kubelet of all worker nodes and kube-apiserver.
+
+## Installation
+
+### Using Jiva operator: 
+
+1. Install openebs local PV provisioner to create hostpath based volumes.
```

Comment:
Please use backticks for code snippets (**everywhere**)

![Screenshot 2021-09-17 at 1 48 31 PM](https://user-images.githubusercontent.com/6524419/133749634-dc4a2424-7a73-4e71-9670-f0a9514eb85e.png)

Reply:
Done